espnet2/tasks/abs_task.py

```diff
@@ -49,7 +49,12 @@
 from espnet2.torch_utils.set_all_random_seed import set_all_random_seed
 from espnet2.train.abs_espnet_model import AbsESPnetModel
 from espnet2.train.class_choices import ClassChoices
-from espnet2.train.dataset import DATA_TYPES, AbsDataset, ESPnetDataset
+from espnet2.train.dataset import (
+    DATA_TYPES,
+    AbsDataset,
+    ESPnetDataset,
+    ESPnetMultiTaskDataset,
```

Comment:
Is there an explanation of `ESPnetMultiTaskDataset`?
Please add it to the appropriate space.

Reply:
Update the docstring at the class. @jctian98 will have more detailed documentation regarding speechlm in his follow up branch.